src/metabase/content_translation/models.clj

```diff
@@ -1,4 +1,4 @@
-(ns metabase-enterprise.content-translation.models
+(ns metabase.content-translation.models
```

Comment:
Something I don't understand - why is it necessary that this be OSS?

Reply:
Thank you, that's a good catch. I'll move everything back into `enterprise`